### DISCUSSION TITLE
Whisper is downloading the wrong model

### DISCUSSION THREAD

**Discussion by svallory** [openai/whisper#2319] [Q&A]
I'm trying to use the `tiny.en` model but whisper downloads the `tiny.pt` model instead. Here's the command I used:

`whisper --model tiny --output_format json --task transcribe --language en --fp16 False ./test-clip.wav`

I'm on a Mac M1 running Sonoma 14.5.

I tried to force it to use `tiny.en` by passing `--model tiny.en` and downloaded a `tiny.en.pt`?! (which is a different file)

**Comment by misutoneko** [comment]
In a nutshell...
`--model tiny    => multilingual model, file name tiny.pt`
`--model tiny.en => english-only model, file name tiny.en.pt`
